shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command.report;
+
+import alluxio.client.block.RetryHandlingBlockMasterClient;
+import alluxio.client.MetaMasterClient;
+import alluxio.client.RetryHandlingMetaMasterClient;
+import alluxio.master.MasterClientConfig;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.wire.BlockMasterInfo;
+import alluxio.wire.MasterInfo;
+
+import java.util.Map;
+
+/**
+ * Print Alluxio cluster summarized information.
+ */
+public class SummaryCommand {
```

Comment:
We should pass in the dependencies for this class so that it can be easily unit tested. The dependencies are the meta master client, block master client, and the PrintStream to write to.

Reply:
Done, thanks

Reply:
It would be good to have a unit test for SummaryCommand which mocks the dependencies so that we can validate the entire print output

Reply:
Oh, I misunderstood this before. 
You mean that It would be better for me to initialize the dependencies in report command and I passed those dependencies to the Summary Command when initializing it.

I will change this part. Thanks

Reply:
@aaudiber 
Would it be better to use PrintSream other than directly use System.out.println ?
It seems to be a little slower and complicated to use printStream to print info to a file and read from that file